tasks/build.py

```diff
@@ -67,6 +67,10 @@ def get_build_env_cfg(cfg):
 
     buildenv = cfg[config.SECTION_BUILDENV]
 
+    job_name = buildenv.get(config.BUILDENV_SETTING_JOB_NAME)
+    log(f"{fn}(): job_name '{job_name}'")
+    config_data = {config.BUILDENV_SETTING_JOB_NAME: job_name}
```

Comment:
This won't work, since `config_data` is overwritten below?

You should change the line 76 below to:

```python
config_data[config.BUILDENV_SETTING_JOBS_BASE_DIR] = jobs_base_dir
```

This also shows this wasn't (re)tested?

Reply:
Yep. Silly me.

Reply:
Fixed. Tested. Should work. 😬